webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  devServer: {
+    contentBase: './dist',
```

Comment:
- [ ] When i tried to start your webserver, I got the following error ⬇️ 
![Screenshot from 2021-08-19 17-10-43](https://user-images.githubusercontent.com/33067256/130083706-f65be3f4-3c72-4e59-8d79-cb0716415b70.png)
When I googled the error this is the [link ](https://stackoverflow.com/questions/67926476/webpack-dev-server-config-contentbase-not-working-in-latest-version) that helped me solve the issue. All I had to to was change from `contentBase` to `static`. Kindly check on this too